code.cpp

```diff
@@ -0,0 +1,136 @@
+#include <array>
+#include <cmath>
+#include <cstdlib>
+#include <iostream>
+#include <unistd.h>
+#include <vector>
```

Comment:
vector is unused. Check also other includes, you can delete them and check compilation